src/config/api.ts

```diff
@@ -1,2 +1 @@
-
-export const apiUrl = process.env.REACT_APP_API_URL ?? 'http://localhost:3001';
\ No newline at end of file
+export const apiUrl = process.env.REACT_APP_API_URL ?? 'http://localhost:3001'
```

Comment:
tu mozemy naprawic prettiera zeby jednak dodawał na końcu sredniki :)

Reply:
mnie też drażni ich brak na końcu wersów. :) jestem za, czekam na opinie innych

Reply:
musimu to poprawić koniecznie!, ale można na to zrobić osobnego taska w sumie bo to będą zmiany dla całej apki :)